src/Microsoft.ComponentDetection.Detectors/spdx/Spdx22ComponentDetector.cs

```diff
@@ -91,7 +91,7 @@ private SpdxComponent ConvertJObjectToSbomComponent(ProcessRequest processReques
             var name = document["name"]?.ToString();
             var spdxVersion = document["spdxVersion"]?.ToString();
 
-            if (rootElements != null && rootElements.Count() > 1)
+            if (rootElements?.Length > 1)
```

Comment:
changing to this syntax builds successfully but does not pass the snapshot checks. Revert it?